### PR TITLE
Feat/api endpoint

### LINKED ISSUE #2: API endpoint
DoD:
- [ ] Implement `POST /upload` API endpoint -  multiform zip archive + meta info (path).

### PR DESCRIPTION
**Pull request by nikitazigman**
closes #2 

* adds `/upload` and `/delete` endpoints 
* adds basic models, schemas, configs, dependencies and settings

### CODE REVIEW

File: cdn_api/cdn_api/api/v1/files.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+@router.post(
+    path="/upload",
+    response_model=Page[responses.FileSchema],
+    summary="Upload a zip archive",
+    description="Upload a zipped files to the CDN",
+    response_description="An information of the uploaded files",
+    status_code=HTTPStatus.CREATED,
+)
+async def upload_files(
+    # _: RateLimiterType,
```
[the remark this answers is not in the document]
Comment (nikitazigman) [reply cdn_api/cdn_api/api/v1/files.py:41]: I am not sure about the rate limiter. The Auth is a valuable feature, similar to the auth integration on the Django admin panel.

File: cdn_api/README.md
[the remark this answers is not in the document]
Comment (nikitazigman) [reply cdn_api/README.md:1]: Yes sure! Will be done :)

File: cdn_api/cdn_api/models/files.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from cdn_api.models.common import (
```
Comment (alena-kono) [inline cdn_api/cdn_api/models/files.py:1]: I think we should add db migrations

**Review by alena-kono — CHANGES_REQUESTED** [review]
Sorry, I've missed one point before

I think we should add db migrations

File: cdn_api/pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    "E",  # pycodestyle
+    "F",  # pyflakes
+    "UP", # pyupgrade,
+    "I",  # isort
+]
+
+[tool.ruff.isort]
+lines-after-imports = 2
+lines-between-types = 1
+section-order = [
```
Comment (alena-kono) [inline cdn_api/pyproject.toml:102]: It's not that important but I've noticed that isort import rules are incorrect as per [PEP8](https://peps.python.org/pep-0008/#imports)

**Review by alena-kono — CHANGES_REQUESTED** [review]

File: cdn_api/cdn_api/models/files.py (hunk 2 above)
Comment (nikitazigman) [reply cdn_api/cdn_api/models/files.py:1]: I was thinking of implementing it in issue #4

**Pull request merged**